components/infobox/wikis/brawlhalla/infobox_team_custom.lua

```diff
@@ -0,0 +1,64 @@
+---
+-- @Liquipedia
+-- wiki=brawlhalla
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Team = require('Module:Infobox/Team')
+local Variables = require('Module:Variables')
+local Class = require('Module:Class')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+
+local CustomTeam = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _team
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	--team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	team.createWidgetInjector = CustomTeam.createWidgetInjector
+	return team:createInfobox(frame)
+end
+
+function CustomTeam:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	return {
+		Cell{
+			name = 'Player number',
+			content = {_team.args.player_number}
+		},
```

Comment:
I'm not saying this is wrong, but it seems weird to me? I'm guessing Smash (used to?) set it, but seems like they have stopped (looking at TL on Smash)

Reply:
currently used on 1 page on brawlhalla
i am indifferent to keeping or removing it

Reply:
I'd say let's remove it. The information is likely to just get forgotten to be updated, and the information is a few centimeters to the left in the Active Roster table.

Reply:
kicked it and all the injector stuff as it was the only thing that was changed by that